modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-connected_{context}"]
+
+= Preparing an agent-based cluster deployment for multicluster engine for Kubernetes (MCE) Operator while connected
```

Comment:
Should Agent be capitalized here? (there are a few lowercase in this file, I think the other files had uppercase)